leap_year.py

```diff
@@ -0,0 +1,18 @@
+def enterYear():
```

Comment:
To calculate, user given year input is leap or not, why do we need three functions in total?